### PR TITLE
Define the DEBUG symbol on Release

### PR DESCRIPTION
**Pull request by victorhurdugaci**
Fixes https://github.com/aspnet/Logging/issues/216

@kichalla @Eilon

### CODE REVIEW

File: src/Microsoft.Framework.Logging.Debug/DebugLogger.cs [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+// We need to define the DEBUG symbol because we want the logger
+// to work even when this package is compiled on release. Otherwise,
+// the call to Debug.WriteLine will not be in the release binary
+#define DEBUG
```
Comment (kichalla) [inline src/Microsoft.Framework.Logging.Debug/DebugLogger.cs]: I believe @Eilon suggestion of having this defined in a separate file using partial class was to limit the scope of the code which sees this symbol defined. You can create a partial class with a private method like `WriteLog` having just the debug writeline statement and the DEBUG defined.

**Comment by victorhurdugaci** [comment]
Updated

**Comment by kichalla** [comment]
:shipit:

**Comment by Eilon** [comment]
Yup, this is the evil trick! :shipit:

**Pull request merged**